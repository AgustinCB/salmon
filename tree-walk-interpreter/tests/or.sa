print "HOLA" or "CHAU";
print nil or "HOLA";
print "HOLA" or false;
print false or "HOLA";
print nil or nil;