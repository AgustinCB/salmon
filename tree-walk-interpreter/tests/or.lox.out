HOLA
HOLA
HOLA
HOLA
Nil
